README.md

```diff
@@ -66,6 +66,14 @@ private void methodRequiresTwoPermission() {
         // Do not have permissions, request them now
         EasyPermissions.requestPermissions(this, getString(R.string.camera_and_location_rationale),
                 RC_CAMERA_AND_LOCATION, perms);
+        // OR for finer control over the rationale dialog
```

Comment:
One last comment:
Developers tend to copy-and-paste so I'd avoid putting both of these in the same snippet as developers will end up with double requests.

I'd say:

```
// old snippet here
```

Or for finer control over the rationale dialog, use `PermissionRequest`:
```
// new snippet here
```